controllers/apirule_controller.go

```diff
@@ -60,13 +58,15 @@ type APIRuleReconciler struct {
 	DefaultDomainName      string
 	Scheme                 *runtime.Scheme
 	Config                 *helpers.Config
+	ReconcilePeriod        time.Duration
+	OnErrorReconcilePeriod time.Duration
 }
 
 const (
 	CONFIGMAP_NAME               = "api-gateway-config"
 	CONFIGMAP_NS                 = "kyma-system"
 	DEFAULT_RECONCILATION_PERIOD = 30 * time.Minute
-	ERROR_RECONCILATION_PERIOD   = time.Minute
+	ERROR_RECONCILATION_PERIOD   = 5 * time.Minute
```

Comment:
I'm not sure about those default periods. Do you think this is OK, or should be longer/shorter?
```suggestion
	ERROR_RECONCILATION_PERIOD   = 5 * time.Minute
```

Reply:
I would go with something like 1 minute.

Reply:
Also typo here `reconciliation`